src/lib.rs

```diff
@@ -385,6 +394,73 @@ impl Interpolate for Quaternion<f32> {
   }
 }
 
+#[cfg(feature = "impl-nalgebra")]
+impl<N : Scalar, D : DimName> Interpolate for na::Point<N, D> 
+  where DefaultAllocator : Allocator<N, D>,
+        <DefaultAllocator as Allocator<N, D>>::Buffer: Copy,
+        N : Interpolate,
+{
+  fn lerp(a: Self, b: Self, t: f32) -> Self {
+    let lerp = |c1 : N, c2 : N| { Interpolate::lerp(c1, c2, t) };
```

Comment:
Please respect conventions from the `splines` project: type ascriptions must be done this way:

    a: N

And not:

    a : N

Reply:
Also, please explain why use a let binding for such a closure.

Reply:
I fixed the type ascriptions issue.

The let binding came from the fact that if I inline it, the line becomes fairly long (102 characters), and I wasn't sure how to reformat a closure as an argument in a clean way. 
A limit of 80 characters doesn't seem like a requirement here, so I can just inline it.